app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,32 @@
+class ReviewsController < ApplicationController
+  def new
+    @item = Item.find(params[:item_id])
+    @review = Review.new
+  end
+
+  def create
+    authenticate_user!
+    @review = Review.new(review_params)
+    @item = Item.find(params[:item_id])
+    @review.item = @item
+    @user = current_user
+    @review.user = @user
```

Comment:
While this works, you can structure it a bit better by using the methods active record provides:

```
item = Item.find(params[:item_id])
@review = item.reviews.build(review_params)
@review.user = current_user
```

Note, if you aren't using a variable in the view, don't make it an instance variable.